Echo/EchoNode.cpp

```diff
@@ -0,0 +1,134 @@
+#include "EchoNode.hpp"
+#include "PolySyncGetOpt.hpp"
+#include <fstream>
+
+using namespace std;
+
+namespace polysync
+{
+
+void PolySyncEcho::initStateEvent()
+{
+    if ( getOpt.wasSingleMsgFiltered() )
```

Comment:
Naming inconsistencies. We've got `msg` in some places and `message` in others. I tend to prefer explicit naming.

Something like `getOpt.singleMessageFilter()`